integration-test/1491-generic-network-fixes.py

```diff
@@ -0,0 +1,4231 @@
+                'surface': u'gravel',
+            }),
+            dsl.relation(1, {
+                'network': u'gr:provincial:73',
+                'ref': u'7312',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+            }, ways=[425300873]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 425300873,
+                'network': u'GR:provincial:73',
+            })
+
+    def test_irfreeways(self):
+        import dsl
+
+        z, x, y = (16, 41219, 25277)
+
+        self.generate_fixtures(
+            dsl.is_in('IR', z, x, y),
+            # https://www.openstreetmap.org/way/549630346
+            dsl.way(549630346, dsl.tile_diagonal(z, x, y), {
+                'highway': u'motorway',
+                'int_ref': u'AH1',
+                'name': u'آزادراه زنجان - تبریز',
+                'oneway': u'yes',
+                'ref': u'2',
+                'source': u'openstreetmap.org',
+            }),
+            dsl.relation(1, {
+                'int_name': u'Asian Highway AH1',
+                'int_ref': u'AH1',
+                'name': u'بزرگراه آسیایی شماره ۱',
+                'name:bn': u'এশিয়ান হাইওয়ে ১',
+                'name:de': u'Asien Fernstraße AH1',
+                'name:en': u'Asian Highway AH1',
+                'name:fa': u'بزرگراه آسیایی شماره ۱',
+                'name:fr': u'Route asiatique AH1',
+                'name:hi': u'एशियाई राजमार्ग १',
+                'name:id': u'Jalan Asia AH1',
+                'name:ja': u'アジアハイウェイ1号線',
+                'name:km': u'ផ្លូវហាយវេអាស៊ី១',
+                'name:ko': u'아시안 하이웨이 1호선',
+                'name:ms': u'Lebuh Raya Asia AH1',
+                'name:my': u'အာရှအဝေးပြေး ၁',
+                'name:ru': u'Азиатский маршрут AH1',
+                'name:th': u'ทางหลวงสายเอเชียAH1',
+                'name:tr': u'Asya Yolu AH1',
+                'name:vi': u'Đường Xuyên Á AH1',
+                'name:zh': u'亚洲公路1号线',
+                'network': u'AH',
+                'ref': u'AH1',
+                'route': u'road',
+                'section': u'Iran',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikidata': u'Q494205',
+                'wikipedia': u'en:AH1',
+            }, ways=[549630346]),
+            dsl.relation(2, {
+                'name': u'آزادراه ۲',
+                'name:en': u'Freeway 2',
+                'network': u'ir:freeways',
+                'ref': u'2',
+                'route': u'road',
+                'source': u'openstreetmap.org',
+                'type': u'route',
+                'wikidata': u'Q4812011',
+                'wikipedia': u'en:Freeway 2 (Iran)',
+            }, ways=[549630346]),
+        )
+
+        self.assert_has_feature(
+            z, x, y, 'roads', {
+                'id': 549630346,
+                'network': u'IR:freeway',
```

Comment:
Should we also be testing AH transform here?

Reply:
Yup. Added test in 87d1555. Looks like they (might?) show the Asian Highway slightly differently in Iran, so I added a transform for that.